CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [0.3.4] - 2023-01-10
```

Comment:
Since it looks like we will now release tomorrow:
```suggestion
## [0.3.4] - 2023-01-11
```

Or we could even consider removing the date of the release entirely from the text here, since github adds the date on the releases page: https://github.com/awslabs/syne-tune/releases and it's present in the git history if anyone wants to view it

Reply:
I'd keep the date in the file, so just looking at the file you see everything, even if you just checked out the repo.